fineract-provider/src/main/java/org/apache/fineract/cob/service/InlineLoanCOBExecutorServiceImpl.java

```diff
@@ -78,6 +79,7 @@
 public class InlineLoanCOBExecutorServiceImpl implements InlineExecutorService<Long> {
 
     private static final String JOB_EXECUTION_FAILED_MESSAGE = "Job execution failed for job with name: ";
+    private static final Integer INLINE_COB_REQUEST_ITEM_LIMIT = 1000;
```

Comment:
Should it be configurable in application.properties?
We might wanna use something similar like `fineract.query.in-clause-parameter-size-limit`?